ccd-definition/CaseEvent.json

```diff
@@ -425,5 +425,31 @@
     "ShowSummary": "N",
     "ShowEventNotes": "Y",
     "EndButtonLabel": "Save and continue"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "migrateCase",
+    "Name": "Migrate Case",
+    "Description": "Migrate Case",
+    "DisplayOrder": 1,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/migration/about-to-submit",
+    "EndButtonLabel": "Save and continue"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "hearingBookingDetails",
+    "Name": "Add hearing details",
+    "Description": "Add hearing booking details to a case",
+    "DisplayOrder": 1,
+    "PreConditionState(s)": "*",
```

Comment:
Is the preConditionState '*'
Correct? 

Not according to this:
Only the HMCTS Admin, Gatekeeper and Judge are able to add and amend hearing details, and they are only able to do this once an application is submitted.

Reply:
👍 correctamundo